src/components/TextField/TextField.scss

```
@@ -138,6 +138,16 @@ $stacking-order: (
       margin: 0;
     }
   }
+
+  &:-webkit-autofill {
+    border-radius: var(--p-border-radius-base, border-radius());
+  }
+
```

```suggestion
// stylelint-disable selector-max-class
```

Looks like you'll need to disable selector max class for this line. That should get the tests to pass